### ISSUE TITLE
Possible to change 404 page and other error image backgrounds?

### ISSUE THREAD

**Issue by bootsie123** [goauthentik/authentik#3974]
Hi! I was wondering if there was a way to change the image that's shown when you encounter a 404 error. Going off of that, is it possible to out right change that default image to something else? Thanks!

**Version and Deployment (please complete the following information):**
 - authentik version: 2022.10.1
 - Deployment: docker-compose

**Comment by Foxite** [comment]
On top of this, I'd also like to be able to change the layout of the 404 page. Perhaps this is best done in the tenant settings.

I'll look into doing this later today, if I remember.

**Comment by Foxite** [comment]
True, but you'll still get an incorrect source link on every page that uses the default background.

**Comment by sevmonster** [comment]
>but you'll still get an incorrect source link on every page that uses the default background

What do you mean? Changing flow background should update all links on flow pages, and you can at least override the default on 404 and non-customizable pages. The latter pages will still have the default `flow_background.jpg` prefetch, but I believe that is planned to be addressed by #4586.

**Comment by Foxite** [comment]
I mean that while updating an individual flow's background will indeed remove the link, if you change the default one by modifying or mounting the builtin file, the link will still be there and I have not yet been able to change that.

**Comment by sevmonster** [comment]
I'm not sure I understand still. If you mount over the file in the container, it's not a problem that "the link will still be there" since it is the correct link.

If you wanted to prevent the prefetch of the image on flow pages while using CSS to change the image, you could set the flow backgrounds to a single pixel image, or a nonexistent URL.

If you want to use specific images for specific pages, or change the image with CSS and not mount over the container, and have the prefetch link match up, you will have to wait on #4586 or use some of the workarounds in #2364.

**Comment by BeryJu** [comment]
I think with link they meant the attribution link to the background at the bottom, which is shown when the background URL starts with `/static`

**Comment by sevmonster** [comment]
I vaguely remember that being a thing, but I have not used the default background in any capacity since I first started using authentik... Still my second suggestion should help there.

**Comment by nicholas-a-guerra** [comment]
Hi team, curious to know if any progress has been made on this? Not sure if I should start digging into backend ways to avoid non uniform backgrounds or if a ui method is on the way. Thanks!

**Comment by codedeeply** [comment]
Seconding the follow up on this; I see this issue linked in a couple newer issues saying setting a default background image per brand/tenant will be implemented "soon," but that was back in 2023. Thanks!
